pkg/sql/schemaexpr/testutils.go

```diff
@@ -32,7 +32,7 @@ func testTableDesc(
 		cols[i] = sqlbase.ColumnDescriptor{
 			Name: columns[i].name,
 			Type: columns[i].typ,
-			ID:   sqlbase.ColumnID(i),
+			ID:   sqlbase.ColumnID(i + 1),
```

Comment:
This change makes the min column ID 1 instead of 0 in these tests to better mimic the "real world" where column IDs are greater than 0.